packages/SqueakMaps-TiledMaps.package/SMATiledMapMorph.class/instance/getBoundingBoxFitting.and..st

```diff
@@ -0,0 +1,35 @@
+helping
+getBoundingBoxFitting: aFirstSMAMapLocationMarker and: aSecondSMAMapLocationMarker
```

Comment:
Umbenennen zu `getBoundingBoxFittingOf: ....` -> liest sich leichter, da man dann gleich ein oder mehrere Parameter erwartet.

Reply:
Da würde ich widersprechen. "Of" ist hier nicht die richtige präposition imo. Der Satz wäre ja "I get a bounding box fitting (the points) A and B. Diesem Muster folgt unsere Funktionssignatur.

Reply:
A right, die Betonung machts ;)